### DISCUSSION TITLE
Difference between stable and closed status in an incident?

### DISCUSSION THREAD

**Discussion by rilutham** [Netflix/dispatch#875] [Q&A] [answered]
What is the difference between stable and closed? When it comes to measuring MTTR, which one should we use (Active > Stable or Active > Closed)?

**Comment by mvilanova** [comment, marked as answer]
We mark incidents as stable when we know the full scope of the incident and the risk/ impact has been mitigated/ remediated. We mark incidents as closed when we have completed a post-incident review and all post-incident action items have been assigned. I'd recommend to use Active -> Stable for Mean time to respond (MTTR).
